src/ristretto.rs

```diff
@@ -937,6 +940,7 @@ impl Debug for RistrettoPoint {
 
 #[cfg(all(test, feature = "stage2_build"))]
 mod test {
+    #[cfg(feature = "rand")]
```

Comment:
Not sure why, but the `rand` crate isn't available during tests unless `std` is enabled. This gating is necessary to make the tests pass with `alloc` alone.